api/api/validator.py

```diff
@@ -21,8 +21,8 @@
 _boolean = re.compile(r'^true$|^false$')
 _dates = re.compile(r'^\d{8}$')
 _empty_boolean = re.compile(r'^$|(^true$|^false$)')
-_group_names = re.compile(r'^(?!^(\.{1,2}|all)$)[A-Za-z0-9.\-_]+$')
-_group_names_or_all = re.compile(r'^(?!^\.{1,2}$)[A-Za-z0-9.\-_]+$')
+_group_names = re.compile(r'^(?!^all$)[A-Za-z0-9\-_]+$')
+_group_names_or_all = re.compile(r'^[A-Za-z0-9\-_]+$')
```

Comment:
Since groups will now be a single file instead of a folder, I've removed the dot `.` from the regex. That is, group names can no longer have a dot in the name.